pkgs/shells/zsh/zsh-fast-syntax-highlighting/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "zsh-fast-syntax-highlighting";
+  version = "2019-07-20";
```

Comment:
According to https://nixos.org/nixpkgs/manual/#sec-package-naming this should read something like "unstable-2019-07-20"
Why use a specific revision if there are (newer) tagged releases? https://github.com/zdharma/fast-syntax-highlighting/releases